lib/Notes/NotesAccordion/components/NotesList/NotesList.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+  FormattedDate,
+} from 'react-intl';
+
+import { MultiColumnList } from '@folio/stripes-components';
+import { noteShape } from '../../NotesAccordion';
+
+
+const COLUMN_NAMES = ['date', 'updatedBy', 'title'];
+const COLUMN_WIDTHS = {
+  date: '30%',
+  updatedBy: '30%',
+  title: '40%',
+};
+
+const columnsMap = {
+  date: <FormattedMessage id="stripes-smart-components.date" />,
+  updatedBy: <FormattedMessage id="stripes-smart-components.updatedBy" />,
+  title: <FormattedMessage id="stripes-smart-components.title" />,
+};
```

Comment:
Depends on just how generic you'd want to make this component - consider setting up the `column` variables - `COLUMN_NAMES`, `COLUMN_WIDTHS` and `columnsMap` as `defaultProps` so it could accept whichever configuration other modules might need to use.